### PR TITLE
kubeone: improve the apply docs

### PR DESCRIPTION
**Pull request by xmudrii**
/assign @kron4eg

### CODE REVIEW

File: content/kubeone/master/using_kubeone/cluster_reconciliation.en.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ terminal and ability to confirm actions, you can use the `--auto-approve`
 flag:
 
 ```bash
-kubeone apply --manifest config.yaml -t . --auto-approve
+kubeone apply --manifest config.yaml -t tf.json --auto-approve
```
Comment (kron4eg) [inline content/kubeone/master/using_kubeone/cluster_reconciliation.en.md:45]: where did `tf.json` came from?

Comment (xmudrii) [reply content/kubeone/master/using_kubeone/cluster_reconciliation.en.md:45]: That’s how we usually call it in all the other documents.

Comment (kron4eg) [reply content/kubeone/master/using_kubeone/cluster_reconciliation.en.md:45]: I meant, in here it's not clear (from previous text) from where this file popup-ed

Comment (xmudrii) [reply content/kubeone/master/using_kubeone/cluster_reconciliation.en.md:52]: Fixed.

**Comment by xmudrii** [comment]
@kron4eg PTAL, I've fixed the comment you left, as well as, added some more information about repairs.

**Pull request merged**